core-bundle/src/Controller/FrontendModule/RootPageDependentModulesController.php

```diff
@@ -37,8 +37,15 @@ public function __invoke(Request $request, ModuleModel $model, string $section,
             return new Response('');
         }
 
+        $moduleModel = $this->container->get('contao.framework')->getAdapter(ModuleModel::class);
+        $module = $moduleModel->findByPk($modules[$pageModel->rootId]);
+
+        $cssID = StringUtil::deserialize($module->cssID, true);
+        $cssID[1] = trim(sprintf('%s %s', $cssID[1], implode(' ', $model->classes)));
+        $module->cssID = $cssID;
```

Comment:
`cssID` in the model is usually serialized AFAIK.

```suggestion
        $module->cssID = serialize($cssID);
```

Reply:
It is, but not needed, https://github.com/contao/contao/blob/4.13/core-bundle/src/Resources/contao/elements/ContentModule.php#L78.

I tried both, makes no difference 😅